skin/src/build.gradle

```diff
@@ -4,7 +4,7 @@ apply plugin: 'me.tatarka.retrolambda'
 apply plugin: 'com.neenbedankt.android-apt'
 apply plugin: 'com.jfrog.bintray'
 
-version = '1.0.0.rc3'
+version = '1.0.0.rc3local'
```

Comment:
Mind changing this .rc3local to .rc3 to fix the merge conflict? Also, looks like skin/build.gradle got dragged into skin/src.